platform-tests/functional-tests/src/test/java/org/ekstep/platform/content/ContentAcceptFlagV3Test.java

```diff
@@ -0,0 +1,378 @@
+package org.ekstep.platform.content;
+
+import com.jayway.restassured.path.json.JsonPath;
+import com.jayway.restassured.response.Response;
+import org.ekstep.platform.domain.BaseTest;
+import org.junit.Test;
+
+import java.io.File;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertEquals;
+
+public class ContentAcceptFlagV3Test extends BaseTest {
+
+    private static final String BASE_PATH = "/content/v3";
+    private static final String DEFAULT_CONTENT_IMAGE_OBJECT_SUFFIX = ".img";
+
+    private static ClassLoader classLoader = ContentRetireV3Test.class.getClassLoader();
+    private static File filePath = new File(classLoader.getResource("UploadFiles/").getFile());
+
+    private final String createDocumentContentRequestBody = "{\"request\": {\"content\": {\"name\": \"Text Book 1\",\"code\": \"test.book.1\",\"mimeType\": \"application/pdf\",\"contentType\":\"Resource\"}}}";
```

Comment:
Please use a fixed prefix for Identifier. e.g: "identifier":"FT_"+<Random Number generated at run time >